### ISSUE TITLE
[Web] [Accessibility] [A11yMAS][Screen Readers][MAS1.3.1][Shortcut key associated with command is not announced by Screen reader]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [Microsoft/monaco-editor#635]
**User Experience:**
User who rely on AT will not get information about the shortcut keys , if shortcut key is not announced by screen reader.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Narrator

**Attachments:**
[MAS1.3.1 Commands.zip](https://github.com/Microsoft/monaco-editor/files/1487132/MAS1.3.1.Commands.zip)

**Repro Steps:**
1.Open the URL: https://microsoft.github.io/monaco-editor/playground.html
2.Navigate to editor and press F1
3. Verify that screen reader do not read the commands associated with the action items in the command palate.

**Current Behavior:**
Screen reader is not announcing the short cut key command provided in the picker with the control name associated with it.

**Expected Behavior:**
Screen reader should announce the shortcut key command along with control which helps AT user to perform the action.

**Note:**
Bug Reference created in Script lab testing : #[712](https://github.com/OfficeDev/script-lab/issues/712)Bug was found while doing Script Lab Testing.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.3.1

**Comment by alexandrudima** [comment]
@bpasero This is why I was thinking the shortcut does not get announced

**Comment by bpasero** [comment]
Duplicate of https://github.com/Microsoft/monaco-editor/issues/657
